superset-frontend/.storybook/preview.jsx

```diff
@@ -23,6 +23,7 @@ import thunk from 'redux-thunk';
 import { Provider } from 'react-redux';
 import reducerIndex from 'spec/helpers/reducerIndex';
 import { GlobalStyles } from '../src/GlobalStyles';
+import { AntdThemeProvider } from '../src/components/AntdThemeProvider';
```

Comment:
@geido this seems to be the way to bring the AntdThemeProvider into all Storybook components... it doesn't seem to cause any issues.